python/tvm/driver/tvmc/transform.py

```diff
@@ -210,9 +229,8 @@ def generate_transform_args(parser):
     # AlterLayout
     parser.add_argument(
         "--desired-layout",
-        choices=["NCHW", "NHWC"],
-        default=None,
-        help="Change the data layout of the whole graph.",
+        nargs="+",
+        help="Change the data/kernel layout of the graph. (i.e. NCHW or NHWC:HWIO)",
```

Comment:
Would be good to also show an example of a list of layouts when `--desired-layout-ops` is provided

Reply:
I have updated the help message.